src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -127,7 +134,7 @@ public OtelInstrumentation()
             }
 
             // Only log duration in HTTP when token is fetched from IDP
-            if (s_durationInHttp.Value.Enabled && authResultMetadata.TokenSource.Equals(TokenSource.IdentityProvider))
+            if (s_durationInHttp.Value.Enabled && tokenSource.Equals(TokenSource.IdentityProvider))
```

Comment:
a.Equals(b) is equivalent to a==b for strings.
Use a.Equals(b, Culture.CaseInsensitiveInvariant) for case insensitive op. But I would think tokenSource will always have the same case so a==b is sufficient?